impeller/renderer/render_target.cc

```diff
@@ -28,6 +28,7 @@ bool RenderTarget::IsValid() const {
     return false;
   }
 
+#ifndef NDEBUG
```

Comment:
This function is actually crazy expensive and I don't think we should be running it any modes.